mne/preprocessing/maxwell.py

```diff
@@ -1516,10 +1517,12 @@ def _overlap_projector(data_int, data_res, corr):
     # Normalize data, then compute orth to get temporal bases. Matrices
     # must have shape (n_samps x effective_rank) when passed into svd
     # computation
-    n = np.sqrt(np.sum(data_int * data_int))
+
+    # we use np.linalg.norm instead of sp.linalg.norm here: ~2x faster!
+    n = np.linalg.norm(data_int)
     Q_int = linalg.qr(_orth_overwrite((data_int / n).T),
                       overwrite_a=True, mode='economic', **check_disable)[0].T
-    n = np.sqrt(np.sum(data_res * data_res))
+    n = np.linalg.norm(data_res)
```

Comment:
I should add the disclaimer that I don't understand why... but good to know
it's not just my system.